### DISCUSSION TITLE
[@types/node] error TS2694: Namespace 'NodeJS' has no exported member 'Global'.

### DISCUSSION THREAD

**Discussion by mrcnski** [DefinitelyTyped/DefinitelyTyped#54294] [Issues with a @types package] [answered]
```
$ tsc
Error: node_modules/@jest/types/build/Config.d.ts(300,38): error TS2694: Namespace 'NodeJS' has no exported member 'Global'.
Error: node_modules/@jest/types/build/Global.d.ts(85,62): error TS2694: Namespace 'NodeJS' has no exported member 'Global'.
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Process completed with exit code 2.
```

[Github Actions job](https://github.com/SkynetLabs/skynet-js/pull/190/checks?check_run_id=2991195979)

**Comment by mrcnski** [comment]
Setting
```json
"skipLibCheck": true
```
in tsconfig.json works, but it would be better to address the root cause of the issue.

Elsewhere on the web:
- https://stackoverflow.com/questions/68262350/how-can-i-resolve-the-typescript-compiler-error-namespace-nodejs-has-no-expor#answer-68262351
- https://stackoverflow.com/questions/68251777/namespace-nodejs-has-no-exported-member-global

**Comment by splincode** [comment, marked as answer]
https://github.com/facebook/jest/pull/11645
